src/components/CsvToBUFRForm.vue

```diff
@@ -96,18 +96,18 @@
                     </v-stepper-window-item>
                     <v-stepper-window-item value="3">
                         <v-card>
-                            <v-card-title>Select topic hierarchy</v-card-title>
-                            <v-card-item>
-                              <TopicHierarchySelector :value="topicSelected" @update:modelValue="newValue => topicSelected = newValue"/>
-                            </v-card-item>
+                            <v-card-title>Select dataset identifier</v-card-title>
+                            <v-col cols="12">
```

Comment:
No, re-using v-card-text for every component, instead of just plain text, is often bad practice because the component very easily overflows. We should be using the Vuetify grid system as much as possible to accurately position components. I'm not going to change every instance of this, only the situations where I found the aforementioned cutoffs.